pysat/tests/test_files.py

```diff
@@ -121,29 +121,30 @@ def test_parse_delimited_filename(self):
         # Note: Can be removed if future instrument that uses delimited
         # filenames is added to routine travis end-to-end testing
         fname = ''.join(('test_{year:4d}_{month:2d}_{day:2d}_{hour:2d}',
-                         '_{min:2d}_{sec:2d}_v01_r02.cdf'))
+                         '_{mins:2d}_{sec:2d}_v01_r02.cdf'))
         year = np.ones(6)*2009
         month = np.ones(6)*12
         day = np.array([12, 15, 17, 19, 22, 24])
         hour = np.array([8, 10, 6, 18, 3, 23])
-        min = np.array([8, 10, 6, 18, 3, 59])
+        mins = np.array([8, 10, 6, 18, 3, 59])
         sec = np.array([58, 11, 26, 2, 18, 59])
         file_list = []
         for i in range(6):
             file_list.append(fname.format(year=year[i].astype(int),
                                           month=month[i].astype(int),
                                           day=day[i], hour=hour[i],
-                                          min=min[i], sec=sec[i]))
-
-        dict = pysat._files.parse_delimited_filenames(file_list, fname, '_')
-        assert np.all(dict['year'] == year)
-        assert np.all(dict['month'] == month)
-        assert np.all(dict['day'] == day)
-        assert np.all(dict['hour'] == hour)
-        assert np.all(dict['min'] == min)
-        assert np.all(dict['day'] == day)
-        assert (dict['version'] is None)
-        assert (dict['revision'] is None)
+                                          mins=mins[i], sec=sec[i]))
```

Comment:
yeah, the mins as a keyword seems weird to me. Open to discussion, of course, as a number of my original style choices have already been updated :)

Reply:
Weird to me too.  :)